esp-lp-hal/Cargo.toml

```diff
@@ -73,15 +73,15 @@ embedded-hal = ["dep:embedded-hal", "dep:embedded-hal-nb"]
 embedded-io = ["dep:embedded-io"]
 
 [[example]]
-name              = "blinky"
+name              = "lp_blinky"
```

Comment:
Renamed because examples are now placed into the same target folder (the workspace's) regardless of the source package.